auxprogs/compileSpliceCands/Makefile

```diff
@@ -1,8 +1,9 @@
 compileSpliceCands : compileSpliceCands.o list.h list.o
-	gcc $(LDFLAGS) -o compileSpliceCands compileSpliceCands.o list.o;
+	$(CC) -o compileSpliceCands compileSpliceCands.o list.o $(CFLAGS) $(LDFLAGS);
```

Comment:
- $(CPPFLAGS) missing?
- why $(CFLAGS)  and $(LDFLAGS) at the line end and not direct after $(CC) like in the Makefiles before
- remove semicolon from end?